packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
+const Button = ({ textColor, children }) => {
+  return <button className={textColor}>{children}</button>;
+};
+<Button textColor="text-green-500">Hello, world</Button>;
+```
 
-- If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
+If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
 
 #### Custom styles
 
 In the event you need to write styles that Tailwind is not able to affectively cover you should try to keep the following guidelines in mind:
 
-- SCSS is available.
-- PurgeCSS does not currently strip styles written outside of `src/styles/tailwind.css`, so it's up to you to keep styles limited to what you're using in your component.
-- Custom styles should be restricted to the component they are being used in. Typically, this is just an `index.scss` file adjacent to your component's `index.tsx` and then imported.
-- Because custom styles, once imported, are included in the global styles you should still be careful about how you're naming classes to avoid collisions. Generally you should match the class name to the name of your component. If you're using a shared component it's recommended to use `[component]-[descriptor]` style (e.g. using a shared UnitRow to display secondary emails could be `unit-row-secondary-emails`).
+- SCSS is available if it's absolutely necessary to add a custom style.
+- Custom styles should be restricted to the component they are being used in. Typically, this is just an imported `index.scss` file adjacent to your component's `index.tsx` and then imported.
+- Because custom styles, once imported, are included in the global styles you should still be careful about how you're naming classes to avoid collisions. Generally you should match the class name to the name of your component and it's recommended to use `[component]-[descriptor]` style (e.g. using a shared UnitRow to display secondary emails could be `unit-row-secondary-emails`). Also, if possible, keep the selector one to two levels deep, never use `!important` to avoid specificity collisions, and don't use `&-` overzealously as it makes classes difficult to search for (one level deep is fine). These recommendations also generally apply to custom component utility classes.
+
+### Reusing components with `fxa-react` and Tailwind
+
+Let's say there's a component in `fxa-admin-panel` that you also want to use in `fxa-settings`. You move this component into `fxa-react` to use it in both `fxa-admin-panel` and `fxa-settings` and style it with Tailwind classes:
```

Comment:
Oh snap you did provide more details for this. Maybe ignore my comment above about why someone might use a shared component.